README.md

```diff
@@ -1,4 +1,4 @@
-![usbkill](https://github.com/pwnsdx/usbkill/blob/master/Resources/USBKill.jpg?raw=true)
+![usbkill](https://github.com/pwnsdx/usbkill/blob/master/Resources/USBKillBanner.jpg?raw=true)
```

Comment:
You can use relative file names. All you need to do is `![usbkill](Resources/USBKillBanner.jpg)`